### PR TITLE
Add navigation and remove meta tag

### PR DESCRIPTION
**Pull request by ididmitrov**
navigation is added with all tags in it nad 1 meta tag is deleted

### CODE REVIEW

**Comment by yankoov** [comment]
Test 123

**Comment by Potrebitel12** [comment]
Comment added by Potrebitel12 - everything is OK.

**Comment by ididmitrov** [comment]
> Comment added by Potrebitel12 - everything is OK.

10x for the review

File: index.html [hunk 1]
```diff
@@ -2,11 +2,16 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
```
Comment (Neyalix) [inline index.html:10]: HTML not working, due to missing meta tag

**Comment by analexsoftuni** [comment]
Ok

File: index.html (hunk 1 above)
Comment (ididmitrov) [reply index.html:10]: My bad, let me fix it

Comment (Potrebitel12) [reply index.html:10]: Are you sure?

**Pull request merged**